src/helpers/analytics.js

```diff
@@ -0,0 +1,10 @@
+export default function dispatchAnalyticsEvent({type, action, ...data}) {
```

Comment:
Can we get some JSDocs on this to describe the shape of the parameter here? Maybe something like
```js
/**
 * Dispatch a custom JavaScript event on document.body for tracking analytics.
 * A separate event listener will be required to record the events to GA.
 * 
 * event.componentName {string} - The name of the component
 * event.action {string} - The action that triggered the analytics event
 * event.details {object} - Contains any number of key-value pairs to further describe the analytics event
 */
```